src/models/User.model.js

```diff
@@ -1,9 +1,26 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const User = sequelize.define(
-  // your code goes here
+  'User',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'users',
+    updatedAt: false,
+    createdAt: false,
```

Comment:
Consider enabling `createdAt` and `updatedAt` timestamps to automatically track when records are created and modified. This can be useful for auditing and historical data analysis.